docs/source/dev_guide/algorithms/svm/support-vector-machine-classifier.rst

```diff
@@ -43,14 +43,18 @@ belongs, the problem is to build a two-class Support Vector Machine
 
 Training Stage
 --------------
-The SVM model is trained using the Boser method [Boser92]_
-reduced to the solution of the quadratic optimization problem
+
+|short_name| provides two methods to train the SVM model:
+
+- Boser method [Boser92]_
```

Comment:
Need to add alternative names for clarification

Reply:
what are the alternative names?

Reply:
I mean SMO (Sequential Minimal Optimization), but some addition clarification about difference between Boser's and Platt's algorithms. I suggest to consult with @PetrovKP about better formulation. He is ready to help with this.

Reply:
I would suggest describing:

```
Boser method [Boser92] which is being decided the Sequential Minimal Optimization algorithm proposed Platt
```

Reply:
added description